src/components/Toaster/ActionItem.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { ENTER_KEY } from '../../constants';
+import styles from './Toaster.scss';
+import { bem } from '../../utils';
+
+export interface ActionProps {
+    /** Action object, containing text, callback and/or href to action  */
+    text: string;
+    callback?: () => void;
+    href?: string;
+}
+
+const { elem } = bem('Toast', styles);
+
+export const ActionItem = (props: { action: ActionProps; dismissToast: () => void }) => {
+    const { action, dismissToast } = props;
+    if (action.callback) {
+        const handleOnClick = () => {
+            action.callback!();
+            dismissToast();
+        };
+        const handleKeyDown = (e) => {
+            if (e.key === ENTER_KEY) {
+                handleOnClick();
+            }
+        };
+        return (
+            <p
+                onClick={handleOnClick}
+                onKeyDown={handleKeyDown}
+                {...elem('toastAction')}
+                role="presentation"
```

Comment:
why did you decide to use `<p role="presentation" />`? 
I'd think it is a button, so should be `button`. Not sure we need to set the role manually, but [presentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) normally mean it is not interactive at all. 
If using `<button />` is complicated in terms of styling it, we can use `<div role="button" />`